examples/thread_duel/README.md

```diff
@@ -0,0 +1,43 @@
+Thread-Duel
+============
+
+This is a thread duel application to show RIOTs multi-threading abilities
```

Comment:
Offline @kaspar030 mentioned this should say it's a show-case of round-robin, and not RIOTs multi-threading since most use cases just follow a priority scheme.